cpp/src/ray/runtime/task/local_mode_task_submitter.cc

```diff
@@ -48,12 +48,15 @@ ObjectID LocalModeTaskSubmitter::Submit(InvocationSpec &invocation,
   std::string task_name =
       invocation.name.empty() ? functionDescriptor->DefaultTaskName() : invocation.name;
 
+  static rpc::JobConfig kDefaultJobConfig;
```

Comment:
Is `static` a premature optimization? Is it possible for this to be called from multiple threads and corrupt the object?